src/gui/log/loglistview.cpp

```diff
@@ -70,13 +70,14 @@ namespace
             painter->save();
             QStyledItemDelegate::paint(painter, option, index); // paints background, focus rect and selection rect
 
-            const QStyle *style = option.widget ? option.widget->style() : QApplication::style();;
+            const QStyle *style = option.widget ? option.widget->style() : QApplication::style();
             const QRect textRect = style->subElementRect(QStyle::SE_ItemViewItemText, &option, option.widget)
                                    .adjusted(1, 0, 0, 0); // shift 1 to avoid text being too close to focus rect
 
-            QFont font = option.font;
-            if (option.font.pointSize() > 0)
-                font.setPointSize(option.font.pointSize()); // somehow this needs to be set directly otherwise painter will use default font
+            QFont font = option.font; // for some reason this doesn't make font a copy of option.font, and we have to manually set font properties
+            if (option.font.pointSizeF() > 0)
+                font.setPointSizeF(option.font.pointSizeF());
+            font.setFamily(option.font.family());
```

Comment:
Personally I would like to place `if` conditionals after "universal" operations (`font.setFamily` here), e.g. I would move this line before `if` block.

Reply:
@Chocobo1 I also fixed comments. Please have a look.